flask_app/blueprints/beams.py

```diff
@@ -13,15 +13,16 @@
 from ..tasks import create_key, beam_up
 
 
-
 beams = Blueprint("beams", __name__, template_folder="templates")
 
+BEAM_RESPONSE = Union[Response, Tuple[str, int]]
```

Comment:
Types are usually named in CamelCase so this should be `BeamResponse`.

Reply:
Also, since you define a type like this in every file you should consider defining it just once and using it everywhere.